compiler/ir/ir.psi2ir/build.gradle.kts

```diff
@@ -8,6 +8,8 @@ dependencies {
     compile(project(":compiler:frontend"))
     compile(project(":compiler:frontend.java"))
     compile(project(":compiler:backend-common"))
+    compile(project(":compiler:ir.backend.common"))
```

Comment:
Would it be possible to get rid of this dependency? Fewer dependencies means faster parallel build, and sometimes cleaner code